locales/it/index.yml

```diff
@@ -2969,6 +2969,11 @@ features:
           singleBtn: "Salva, stampa o condividi"
           open: "Apri"
           save: "Salva"
+          savedAtLocation: "Il documento {{name}} e' stato salvato nella cartella Scaricati ({{location}})."
```

Comment:
I wouldn't show to users the full path of the saved document (it's a system folder). Maybe saying "... was successfully saved in your Download folder" could be enough IMHO.

Let's wait for @thisisjp's feedback on this.

Reply:
I agree: is it possibile to refer to a "Download" folder? I mean, is this how the feature works on Android?

Reply:
It would be fine to say _Download folder_ (or _cartella Scaricati_ in this specific example).
The downloaded files are usually conveyed to a _Download center_ and their availability is signalled via the OS top bar. Then they can be accessed via Files app.